documentation/configuring_bolt.md

```diff
@@ -1,8 +1,41 @@
 # Configuring Bolt
 
-You can configure Bolt's options and features at a project level, a user level,
-or a system-wide level. Unless your use case requires setting user-specific or
-system-wide configurations, configure Bolt at the project level. 
+Bolt has many options and features that can be configured to suit your
+project's needs. In general, Bolt configuration falls into four categories:
+
+- **Bolt behavior**
+
+  Configure how Bolt itself runs, such as the format to use when displaying
+  output or how many threads to use when connecting to targets.
+
+- **Projects**
+
+  Configure the Bolt project that you are running in, such as paths to the
+  inventory file or a Hiera configuration file.
+
+- **Transports**
+
+  Configure the transports that Bolt uses to connect to targets, such as
+  the path to a private key when using SSH or the port to connect to when
+  using WinRM.
+
+- **Inventory data**
+
+  Group and configure the targets that you connect to and run commands on with
+  Bolt.
+
+Each type of configuration can be set in a different file. You can configure
+Bolt's options and features at a project level, a user level, or a system-wide
+level. Unless your use case requires setting user-specific or system-wide
+configurations, configure Bolt at the project level.
+
+| Type of Configuration | [`inventory.yaml`](bolt_inventory_reference.md) | [`bolt-project.yaml`](bolt_project_reference.md) | [`bolt-defaults.yaml`](bolt_defaults_reference.md) |
+| --- | :-: | :-: | :-: |
```

Comment:
Good move leaving bolt.yaml out of the table. I think the less we highlight it, the better.